Ginger/Ginger/ALM/ALMIntegration.cs

```diff
@@ -651,12 +651,12 @@ public void OpenALMItemsFieldsPage()
             AlmRepo.OpenALMItemsFieldsPage();
         }
 
-        public void ALMDefectsProfilesPage()
-        {
-            GingerCoreNET.ALMLib.ALMConfig AlmConfig = GetDefaultAlmConfig();
-            UpdateALMType(AlmConfig.AlmType);
-            AlmRepo.ALMDefectsProfilesPage();
-        }
+        //public void ALMDefectsProfilesPage()
+        //{
```

Comment:
Codacy found an issue: [Remove this commented out code.](https://app.codacy.com/manual/Ginger-Automation/Ginger/pullRequest?prid=5717441)